TheraLang.BLL/Services/ProjectParticipationService.cs

```diff
@@ -62,22 +62,24 @@ public IEnumerable<ProjectParticipationDto> GetAll()
                 .ForMember(m => m.RequestedUserName, opt => opt.MapFrom(m => m.User.UserName))
                 .ForMember(m => m.RequestedUserEmail, opt => opt.MapFrom(m => m.User.Email))
             ).CreateMapper();
-            var projectParticipationDtos = mapper.Map<IEnumerable<ProjectParticipation>, IEnumerable<ProjectParticipationDto>>(projectParticipations);
+            var projectParticipationDtos =
+                mapper.Map<IEnumerable<ProjectParticipation>, IEnumerable<ProjectParticipationDto>>(
+                    projectParticipations);
 
             return projectParticipationDtos;
         }
 
 
-        public async Task CreateRequest(Guid userId, int projectId)
+        public async Task CreateRequestAsync(Guid userId, int projectId)
         {
             try
             {
-                var user = _unitOfWork.Repository<User>().Get().Where(u => u.Id == userId).FirstOrDefault();
-                var isRequested = _unitOfWork.Repository<ProjectParticipation>()
-                    .Get()
-                    .Any(p => p.ProjectId == projectId && p.User.Id == userId);
+                var user = await _unitOfWork.Repository<User>().GetAsync(u => u.Id == userId);
+                var isRequested = await _unitOfWork.Repository<ProjectParticipation>()
+                    .GetAsync(p => p.ProjectId == projectId && p.User.Id == userId);
+
 
-                if (!isRequested)
+                if (isRequested == null)
```

Comment:
this makes no sense to me, you call get and waiting `isRequested` named as booolean